libs/shared/utils/src/lib/answer-main.ts

```diff
@@ -4,11 +4,10 @@
  */
 
 import { ipcRenderer } from "electron-better-ipc"
-import { CallRendererEvent } from "device/adapters"
 
-export const answerMain = (
-  event: CallRendererEvent,
-  callback: (data: unknown) => void | PromiseLike<void>
+export const answerMain = <DataType = unknown>(
+  event: string,
```

Comment:
why did you remove CallRendererEvent type?

Reply:
I did it this way because I wanted to provide more flexibility in defining events. Using a string instead of CallRendererEvent allows for easier adaptation and greater universality of the function. I thought this would make things easier for various applications, without limiting ourselves to pre-defined event types. It was about making things simpler and more flexible.

Reply:
https://github.com/mudita/mudita-center/pull/1765/commits/cad0360218a266aaa30ba5a1a8386855c6b65a43